pypeit/wavetilts.py

```diff
@@ -188,34 +188,41 @@ def show(self, waveimg=None, wcs_match=True, in_ginga=True, show_traces=False):
                 If True, show the traces of the tilts on the image.
 
         """
+        # get tilt_img_dict
+        if (Path(self.calib_dir).resolve() / self.tiltimg_filename).exists():
+            tilt_img_dict = buildimage.TiltImage.from_file(Path(self.calib_dir).resolve() / self.tiltimg_filename)
+        else:
+            msgs.error(f'Tilt image {str((Path(self.calib_dir).resolve() / self.tiltimg_filename))} NOT FOUND.')
+
         # get slits
+        slitmask = None
         if (Path(self.calib_dir).resolve() / self.slits_filename).exists():
             slits = slittrace.SlitTraceSet.from_file(Path(self.calib_dir).resolve() / self.slits_filename)
-            slitmask = slits.slit_img(initial=True, flexure=self.spat_flexure)
-            left, right, mask = slits.select_edges(flexure=self.spat_flexure)
-            gpm = mask == 0
+            _slitmask = slits.slit_img(initial=True, flexure=self.spat_flexure)
+            _left, _right, _mask = slits.select_edges(flexure=self.spat_flexure)
```

Comment:
I think you may need to resize _mask as well. This should probably crash on x-shooter. Did you check. You can resize mask with the arc.resize_mask2arc

Reply:
@jhennawi yes I checked with x-shooter and some other instrument and it works fine. `_mask` does not need to be resized because it's a 1D array with a size equal to the number of slits, i.e., it has only one value per slit.

Reply:
Got it. I was confused.